small/lsregion_asan.c

```diff
@@ -30,18 +30,49 @@
  */
 #include "lsregion.h"
 
+SMALL_NO_SANITIZE_ADDRESS static void *
+alloc_reserve(struct lsregion *lsregion, size_t size, size_t alignment)
+{
+	struct lsregion_allocation *alloc = NULL;
+	if (!rlist_empty(&lsregion->allocations)) {
+		alloc = rlist_last_entry(&lsregion->allocations, typeof(*alloc),
+					 link);
+		if (alloc->id == LSLAB_NOT_USED_ID) {
+			if (alloc->size >= size) {
+				return alloc;
+			} else {
+				rlist_del_entry(alloc, link);
+				small_asan_free(alloc);
+			}
+		}
+	}
+	alloc = small_asan_alloc(size, alignment,
+				 sizeof(struct lsregion_allocation));
+	alloc->size = size;
+	alloc->used = 0;
+	alloc->id = LSLAB_NOT_USED_ID;
+	rlist_add_tail_entry(&lsregion->allocations, alloc, link);
+	return alloc;
+}
+
+SMALL_NO_SANITIZE_ADDRESS void *
+lsregion_aligned_reserve(struct lsregion *lsregion, size_t size,
```

Comment:
Looks like we don't need `SMALL_NO_SANITIZE_ADDRESS` here.

Reply:
Fixed. Thanks!